includes/Checker/Experimental_Check.php

```diff
@@ -0,0 +1,24 @@
+<?php
+/**
+ * Trait WordPress\Plugin_Check\Checker\Experimental_Check
+ *
+ * @package plugin-check
+ */
+
+namespace WordPress\Plugin_Check\Checker;
+
+/**
+ * Trait for stable checks.
+ *
+ * @since n.e.x.t
+ */
+trait Experimental_Check {
+	/**
+	 * Returns the checks stability
+	 *
+	 * @return int
+	 */
```

Comment:
Please update the docs based on my other comment in the interface.

Reply:
This has been updated